server/config/brain/humidity-sensor/answers.en.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "label": "humidity-sensor.get-in-room.success",
+    "answers": ["It's {{ humidity }} {{ unit }} in the {{ roomName }}."]
+  },
+  {
+    "label": "humidity-sensor.get-in-room.fail.no-results",
+    "answers": ["No humidity values were recorded in the last 12 hours in this room."]
```

Comment:
I think in the code you're getting the last hour humidity, no?

Reply:
Yep fixed ( temperature answers need to be fixed too )